package-lock.json

```diff
@@ -1,8 +1,5343 @@
 {
   "name": "service-runner",
-  "version": "3.0.0",
-  "lockfileVersion": 1,
```

Comment:
Unintended change? This may confuse Node <= 10

Reply:
Definitely unintended. I don't even know what that does. 

To be honest, I am a bit wondering why we commit this file in the git repo. I understand why we would that for anything that is to be deployed, but does this argument also stand for a library (which service-runner is)?

Anyway, I 'll amend.

Reply:
I think this is still important for reproducible builds e.g. in CI and during publish. Otherwise, we may get different resolutions for each run